bodo/pandas/base.py

```diff
@@ -443,6 +446,100 @@ def to_datetime(
     )
 
 
+@check_args_fallback(unsupported="all")
```

Comment:
`unsupported="all"` ignores the first argument?

Reply:
check_args_fallback only applies to arguments with default values.